docs/spec/spec.md

```diff
@@ -0,0 +1,309 @@
+| channelable | Channelable | Holds a list of downstream subscribers for the channel. | |
+| eventTypes |[]String| An array of event types that will be passed on the Channel. | Must be objects with kind:EventType. |
+
+\*: Required
+
+#### Metadata
+
+##### Owner References
+
+ * If the Pipeline controller created this Channel: Owned by the originating Pipeline.
+ * Owned by the Provisioner used to provision the Channel.
+
+#### Status
+
+| Field | Type | Description | Limitations |
+| --- | --- | --- | --- |
+| Sinkable | Sinkable | Address to the endpoint as top-level domain that will distribute traffic over the provided targets from inside the cluster. | |
+| subscribable | Subscribable | | |
+| Conditions | Conditions | Standard Subscriptions| |
+
+##### Conditions
+
+ * **Ready.** True when the Channel is provisioned and ready to accept events.
+ * **Provisioned.** True when the Channel has been provisioned by a controller.
+
+#### Events
+
+ * Provisioned
+ * Deprovisioned
+
+### Life Cycle
+
+| Action | Reactions | Limitations |
+| --- | --- | --- |
+| Create | The Provisioner referenced will take ownership of the Channel and begin provisioning the backing resources required for the Channel depending on implementation. | Only one Provisioner is allowed to be the Owner for a given Channel. |
+| Update | The Provisioner will synchronize the Channel backing resources to reflect the update. | |
+| Delete | The Provisioner will deprovision the backing resources if no longer required depending on implementation. | |
+
+---
+
+## kind: Provisioner
+
+### group: eventing.knative.dev/v1alpha1
+
+_Describes an abstract configuration of a Source system which produces events
+or a Channel system that receives and delivers events._
+
+### Object Schema
+
+#### Spec
+
+| Field | Type | Description | Limitations |
+| --- | --- | --- | --- |
+| type*| TypeMeta<sup>1</sup> | The type of the resource to be provisioned. | Must be Source or Channel. |
+
+\*: Required
+
+1: Kubernetes type.
+
+#### Metadata
+
+##### Owner References
+
+ * Owns EventTypes.
+
+#### Status
+
+| Field | Type | Description | Limitations |
+| --- | --- | --- | --- |
+| provisioned |[]ProvisionedObjectStatus| Status of creation or adoption of each EventType and errors therein. | It is expected that a provisioner list all produced EventTypes, if applicable. |
+
+#### Events
+
+ * Source created
+ * Source deleted
+ * Event types installed
+
+### Life Cycle
+
+| Action | Reactions | Limitations |
+| --- | --- | --- |
+| Create | Creates and owns EventTypes produced, or adds Owner ref to existing EventTypes. | Verifies Json Schema provided by existing EventTypes; Not allowed to edit EventType if previously Owned; |
+| Update | Synchronizes EventTypes. | |
+| Delete | Removes Owner ref from EventTypes. | |
+
+---
+
+## kind: Subscription
+
+### group: eventing.knative.dev/v1alpha1
+
+_Describes a direct linkage between an event publisher and an action._
```

Comment:
The Subscription links from a Channel to a Callable and/or Sinkable, and a Channel's primary benefit is to decouple event publishers from event subscribers... so any linkage between them is *indirect*. The "event publisher" shouldn't be mentioned within the context of Subscription since it's on the other side of the Channel. In fact there can be a many:many relationship between event publishers and Channels.

Reply:
Updated to:

`_Describes a linkage between a Subscribable and a Targetable and/or Sinkable._`